Remove inert-svg-hittest.html from Interop 2022

nt1m opened the pull request:
It actually passes on WebKit test runner: https://github.com/WebKit/WebKit/blob/5df171bce18e37178986ed7e475a0db6eef0b814/LayoutTests/imported/w3c/web-platform-tests/html/semantics/interactive-elements/the-dialog-element/inert-svg-hittest-expected.txt

So this is most likely a WebDriver bug. Given it passes in every browser, I don't see much point in bringing down the score here.

<!--  Thanks for sending a pull request!  Here are some tips for you:

1. If this is your first time, please read our README.md: https://github.com/web-platform-tests/wpt-metadata/blob/master/README.md 
2. Please label this pull request `do not merge yet` upon creation because the auto-merge feature is enabled in this repository. If this pull request is finished, feel free to assign foolip/kyleju as reviewers, or we will review and merge them automatically.

foolip commented:
@nt1m did you file an issue on https://github.com/web-platform-tests/interop-2022/issues for this change, or rather is it covered by one of the issues you filed recently?

It seems unfortunate to drop the test without understanding why it's failing on wpt.fyi.

nt1m commented:
> @nt1m did you file an issue on https://github.com/web-platform-tests/interop-2022/issues for this change, or rather is it covered by one of the issues you filed recently?
> 
> It seems unfortunate to drop the test without understanding why it's failing on wpt.fyi.

I added this test to WPT & WebKit in https://bugs.webkit.org/show_bug.cgi?id=235836, I am fairly sure this doesn't find any significant interop issue, given WebKit passes the test.

The Safari WebDriver error on wpt.fyi is:

```
ERROR message: Traceback (most recent call last):
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/executorwebdriver.py", line 420, in run_func
    self.result = True, self.func(self.protocol, self.url, self.timeout)
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/executorwebdriver.py", line 526, in do_testharness
    done, rv = handler(result)
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/base.py", line 724, in __call__
    return callback(url, payload)
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/base.py", line 740, in process_action
    result = action_handler(payload)
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/actions.py", line 87, in __call__
    self.protocol.action_sequence.send_actions({"actions": actions})
  File "/Users/runner/work/1/s/tools/wptrunner/wptrunner/executors/executorwebdriver.py", line 238, in send_actions
    self.webdriver.actions.perform(actions['actions'])
  File "/Users/runner/work/1/s/tools/webdriver/webdriver/client.py", line 22, in inner
    return func(self, *args, **kwargs)
  File "/Users/runner/work/1/s/tools/webdriver/webdriver/client.py", line 283, in perform
    actions = self.session.send_session_command("POST", "actions", body)
  File "/Users/runner/work/1/s/tools/webdriver/webdriver/client.py", line 661, in send_session_command
    return self.send_command(method, url, body, timeout)
  File "/Users/runner/work/1/s/tools/webdriver/webdriver/client.py", line 625, in send_command
    raise err
webdriver.error.WebDriverException: element not interactable (400): 
```

nt1m commented:
To me this seems like a Safari WebDriver issue rather than a WebKit one.

foolip commented:
@nt1m is there a public bug for that which could be linked from the wpt.fyi failure?

I would have liked this change to go through the usual review, but it's on the agenda for today so there'll be a chance for folks to object.

nt1m commented:
> @nt1m is there a public bug for that which could be linked from the wpt.fyi failure?

WebDriver failures are only tracked internally.

foolip commented:
@mfreed7 @josepharhar can either of you help take a look at this test and say if you think it's testing something that is important to cover?

Also ping @jgraham for Gecko position.

nt1m commented:
It's testing that SVG elements can't be hit-tested when made inert by a modal dialog, it was a bug someone found in WebKit, which I've fixed and added a test for: https://bugs.webkit.org/show_bug.cgi?id=235836 . This is more of a regression test than an interop test fwiw.

nt1m commented:
One way I could rewrite it to avoid the WebDriver exception is changing:
```
    await new test_driver.Actions()
        .pointerMove(0, 0, { origin: wrapper })
        .pointerDown()
        .send();
```

to something like: 

```
    let wrapperRect = wrapper.getBoundingClientRect();
    await new test_driver.Actions()
        .pointerMove(wrapperRect.x + 1, wrapperRect.y + 1, { origin: "viewport" })
        .pointerDown()
        .send();
```

nt1m commented:
Ah nevermind me, it might actually the cleanup function that's broken.

Anyway trying to work around it here: https://github.com/web-platform-tests/wpt/pull/35414

mfreed7 commented:
The fix at https://github.com/web-platform-tests/wpt/pull/35414 looks good to me - thanks for taking a look at this.

nt1m commented:
I re-added the test in Interop 2022: https://github.com/web-platform-tests/wpt-metadata/pull/2946 

Sorry for doing this change without notice.
